plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -642,6 +642,29 @@ items:
     - serviceaccounts
     verbs:
     - update
+  - apiGroups:
+    - apps
+    resources:
+    - controllerrevisions
+    - daemonsets
+    - deployments
+    - deployments/rollback
+    - deployments/scale
+    - deployments/status
+    - replicasets
+    - replicasets/scale
+    - replicasets/status
+    - statefulsets
+    - statefulsets/scale
+    verbs:
```

Comment:
it seems all verbs included. is there a wildcard for that purpose?

Reply:
kube-controller-manager need all read-write access for apps group, that's why all verbs included.